QUEST_LV_0200.tsv

```diff
@@ -5591,57 +5591,57 @@ QUEST_LV_0200_20150323_005590	Upham was seperated from the colleagues due to the
 QUEST_LV_0200_20150323_005591	You've took Upham to Broknews. Talk with Broknews.
 QUEST_LV_0200_20150401_005592	How would we know about the mind of the baron.{nl}He told us to behave well when the revelator comes so we are just doing the work he told us to do.{nl}
 QUEST_LV_0200_20150401_005593	Well, if you see the baron yourself, you may find out something.{nl}He is working on something big at Mirukiti farm, so why don't you go meet him?
-QUEST_LV_0200_20150401_005594	I have to tie the sacks of the grains, but I don't have enough strings.{nl}I am sure that the monsters near here took them. Please prepare tight strings for me.
+QUEST_LV_0200_20150401_005594	$I have to tie the sacks of the grains, but I don't have enough rope.{nl}I am sure that the monsters near here took them. Please prepare tight rope for me.
 QUEST_LV_0200_20150401_005595	Ah, ignore what the workers near here say.{nl}They are whining.
 QUEST_LV_0200_20150401_005596	Good. {nl}Now the task here is completed.
 QUEST_LV_0200_20150401_005597	Hey, Are you the revelator?{nl}Then I welcome you. The baron told me to treat you well.{nl}
 QUEST_LV_0200_20150401_005598	There is a small task you can do.{nl}We don't have enough workforce here. Ah, I am not talking about farming.{nl}
 QUEST_LV_0200_20150401_005599	Something seems to be moving in the sacks.
-QUEST_LV_0200_20150401_005600	I want to request to you to finish the rest.{nl}There are some abondoned sacks on Tvora Highway. I want you to complete them.
+QUEST_LV_0200_20150401_005600	$I want to request to you to finish the rest.{nl}There are some abandoned sacks on Tvora Road. I want you to complete them.
```

Comment:
"I request that you finish the rest.{nl}There are some abandoned sacks on Tvora Road. I want you to complete them."
or
"I'd like to request you to finish the rest.{nl}There are some abandoned sacks on Tvora Road. I want you to complete them."
or just remove the "to" after "request".